public/js/reducers/management.js

```diff
@@ -1,31 +1,47 @@
 import * as actionTypes from 'constants/action-types';
 
 
-export default function management(state, action) {
-  console.log('management store: got action', action);
+export const initialMgmtState = {
+  error: null,
+  tab: null,
```

Comment:
do we not have a default tab? Oh, I guess this will change once we move to the non-accordion layout.